cdap-ui/app/directives/datetime-range/datetime-range.html

```diff
@@ -0,0 +1,61 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="datetime-range"
+     ng-class="{ 'open': RangeController.startTimeOpen || RangeController.endTimeOpen }">
+  <div class="date-text-input text-center">
+    <div class="start-time datetime-input">
+      <input type="text"
+             class="form-control"
+             placeholder="Start Time"
+             ng-model="dateRange.startTime"
+             ng-click="RangeController.openStartTime()"
+             ng-class="{ 'active': RangeController.startTimeOpen }">
+    </div>
+    <span>to</span>
+    <div class="end-time datetime-input">
+      <input type="text"
+             class="form-control"
+             placeholder="End Time"
+             ng-model="dateRange.endTime"
+             ng-click="RangeController.openEndTime()"
+             ng-class="{ 'active': RangeController.endTimeOpen }">
+    </div>
+  </div>
+
+  <div>
+    <my-datetime-picker
+      date-data="dateRange.startTime"
```

Comment:
@elfenheart In angular there is a general convention to have `data-<scope-variable>` and angular ignores the prefix. The assumption is that any html attribute that prefixes with `data-` should be considered as user parameter. This `date-data` is a little confusing in that sense. We might want to rename this a little different. Its subjective and if you feel its fine I am ok with it.